Composer/packages/extensions/adaptive-form/src/components/ErrorInfo.tsx

```diff
@@ -1,25 +1,28 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 import React from 'react';
+import formatMessage from 'format-message';
 import { FallbackProps } from 'react-error-boundary';
 import { MessageBar, MessageBarType } from 'office-ui-fabric-react/lib/MessageBar';
 
 const ErrorInfo: React.FC<FallbackProps> = ({ componentStack, error }) => (
   <div style={{ marginRight: '20px' }}>
     <MessageBar isMultiline={false} messageBarType={MessageBarType.error}>
       <p style={{ whiteSpace: 'pre-wrap' }}>
-        <strong>Oops! An error occured in the form editor!</strong>
-        <br />
-        This is likely due to malformed data or missing functionality in Composer.
-        <br />
-        Try navigating to another node in the visual editor.
+        {formatMessage.rich(
+          '<strong>Oops! An error occured in the form editor!</strong><br />This is likely due to malformed data or missing functionality in Composer.<br />Try navigating to another node in the visual editor.',
```

Comment:
It seems like it would be pretty easy for a localizer to accidentally mess up the HTML here.  I would consider doing 3 format messages here.

Reply:
This looks like a good use for that "id" argument so a translator knows they go together, too.